.github/actions/run-backend-tests/action.yml

```diff
@@ -69,6 +69,11 @@ runs:
               python -m pip install -r requirements-dev.txt
               python -m pip install -r requirements.txt
 
+        - name: Install reportlog
+          shell: bash
+          run: |
+              python -m pip install pytest-reportlog
```

Comment:
I think this could be added to `requirements-dev.txt` and then it'd be installed by line 69 above?